content/open-access.md

```diff
@@ -0,0 +1,55 @@
+
+Traditionally, success in scholarly research is measured by impact factor - a researcher is often deemed more successful and advances more quickly in their career if they publish in journals with high impact factors (the "publish or perish" mantra in research). However, many of these high impact factor journals are subscription based, meaning that the research is locked behind a paywall with high fees associated with accessing it or high costs to the researcher for making it open access upon publication.
+
+In academia, there is widespread dissatisfaction with this expensive traditional publishing model which is unsustainable and redundant. In the current publishing landscape, Universities, research institutions and funding agencies are often double paying for the research they produce - they have to pay the high journal subscription costs in order to access research articles, for which they have already funded the research and publication of. Meanwhile, journal costs continue to increase while library budgets continue to decrease, and are already unaffordable to smaller research institutes and underrepresented communities.
+
+The [Budapest Open Access Initiative (BOAI)](http://www.budapestopenaccessinitiative.org/) was formed in response to the growing demand to make research free and available to anyone with a computer and an internet connection - the research process hinges on ensuring fast and widespread access to research findings such that all communities have the opportunity to build upon them and participate in scholarly conversations. The BOAI was the first initiative to use the term "Open Access", which it defined on February 14, 2002 as follows:
+>By "open access" to [peer-reviewed research literature], we mean its free availability on the public internet, permitting any users to read, download, copy, distribute, print, search, or link to the full texts of these articles, crawl them for indexing, pass them as data to software, or use them for any other lawful purpose, without financial, legal, or technical barriers other than those inseparable from gaining access to the internet itself. The only constraint on reproduction and distribution, and the only role for copyright in this domain, should be to give authors control over the integrity of their work and the right to be properly acknowledged and cited.
+
+Further major drivers behind Open Access relate to equality within and between communities, as well as bridging the global North-South research divide (see [The academic, economic and societal impacts of Open Access: an evidence-based review](https://f1000research.com/articles/5-632/v3) for more information on the history of Open Access).
+
+Open Access has been demonstrated to benefit researchers, science and society. By making their research outputs openly available, researchers and research institutions can benefit from, for example:
```

Comment:
I think this list and the next are captured in the image, so might be better to remove them from the text and include a sentence along the lines of

> Some of the benefits of Open Access are highlighted in the figure below.